icu4c/source/test/intltest/numbertest.h

```diff
@@ -88,7 +88,7 @@ class NumberFormatterApiTest : public IntlTestWithFieldPosition {
     CurrencyUnit GBP;
     CurrencyUnit CZK;
     CurrencyUnit CAD;
-    CurrencyUnit ESP;
+    CurrencyUnit __ESP;
```

Comment:
Wouldn't it be better to #undef ESP on Solaris?

Reply:
Or, unconditionally:
```
// Solaris #defines ESP.
#ifdef ESP
#   undef ESP
#endif
```

Reply:
Depending on what ESP is, `__ESP` may be better if the compiler treats it specially. I'm inclined to merge this as is unless you object.

Reply:
The C standard reserves identifiers that begin with two underscores, so I think `__ESP` is not just ugly but worse than before. I prefer the `#undef`; and as Andy says, that would be self-documenting.

Reply:
Good point.  OK, let's go with the `#ifdef` … `#undef`.  

@tropikhajma please update, but note that this may miss ICU 64 because we are post deadline